akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -542,6 +542,25 @@ trait FlowOps[+Out, +Mat] {
   def scan[T](zero: T)(f: (T, Out) ⇒ T): Repr[T, Mat] = andThen(Scan(zero, f.asInstanceOf[(Any, Any) ⇒ Any]))
 
   /**
+   * Similar to `scan` but only emits its result when the upstream completes,
+   * after which it also completes. Applies the given function towards its current and next value,
+   * yielding the next current value.
+   *
+   * If the function `f` throws an exception and the supervision decision is
+   * [[akka.stream.Supervision.Restart]] current value starts at `zero` again
+   * the stream will continue.
+   *
+   * '''Emits when''' the function scanning the element returns a new element
```

Comment:
This is the one: https://github.com/akka/akka/blob/release-2.3-dev/akka-docs-dev/rst/stages-overview.rst